File: Rotator/RotatorSlideShow/RotatorSlideShowCS/RotatorSlideShowFile.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Text;
using RotatorSlideShow.Properties;

namespace RotatorSlideShow
{
    [Serializable]
    public class RotatorSlideShowFile
    {
        ArrayList files;

        public RotatorSlideShowFile()
        {
            files = new ArrayList();
        }

        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
        public ArrayList Files
        {
            get { return files; }
        }
    }
}

File: Rotator/RotatorSlideShow/RotatorSlideShowVB/RotatorSlideShowFile.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Text
Imports RotatorSlideShow.Properties

Namespace RotatorSlideShow
	<Serializable> _
	Public Class RotatorSlideShowFile
		Private files_Renamed As ArrayList

		Public Sub New()
			files_Renamed = New ArrayList()
		End Sub

		<DesignerSerializationVisibility(DesignerSerializationVisibility.Content)> _
		Public ReadOnly Property Files() As ArrayList
			Get
				Return files_Renamed
			End Get
		End Property
	End Class
End Namespace